README.md

```diff
@@ -55,6 +55,8 @@ If you'd like to license the source for commercial use, [please reach out](https
 
 ## Questions
 
+- **Where can I find out why a particular exception has been added?** We mostly do two kinds of mitigations -- either allowing [certain trackers](features/tracker-allowlist.json) or disabling one or more [protections](features/) on specific URLs or on certain [platforms](overrides). If you search in Code for the domain you're interested in, in most cases there will be a `"reason"` key below the exception for it with the URL of either an issue (old process) or a pull request (current process). Within the description of that linked URL we explain why we've added a breakage mitigation to the site in question and exactly what we're allowing or disabling to improve users' experience on the page.
```

Comment:
@laghee have you been able to verify these links work (specifically protections and platforms)? Just want to be sure before we merge.

Reply:
Ehrm, no actually 😬 They look... weirdly specific to my branch, but I was hoping that was just an in-flux formatting artifact. I could replace with full URLs -- maybe that's more foolproof?

Reply:
I suppose let's use full urls to be safe 👍

Reply:
Updated!